from ScannerApp.logger import logger


logger.warning(f'Unexpected Error {1/2} %s', 'another line')

